### PR TITLE
feat: set room with state

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

File: frontend/containers/DocumentEdit/DocumentEdit.jsx [hunk 1]
```diff
@@ -143,13 +133,24 @@ class DocumentEdit extends Component {
     this.currentCollection = currentCollection
   }
 
+  handleRoomChange() {
+    const {state, actions, documentId} = this.props
```
Comment (eduardoboucas) [inline frontend/containers/DocumentEdit/DocumentEdit.jsx]: Could you add [a space after initialisation](https://github.com/dadi/publish/blob/master/docs/coding-guidelines.md#white-space)?

File: frontend/containers/DocumentEdit/DocumentEdit.jsx [hunk 2]
```diff
@@ -224,7 +226,9 @@ class DocumentEdit extends Component {
   }
 
   componentWillUnmount() {
+    const {actions} = this.props
```
Comment (eduardoboucas) [inline frontend/containers/DocumentEdit/DocumentEdit.jsx]: Could you add [a space after initialisation](https://github.com/dadi/publish/blob/master/docs/coding-guidelines.md#white-space)?

File: frontend/containers/DocumentEdit/DocumentEdit.jsx [hunk 3]
```diff
@@ -143,13 +133,20 @@ class DocumentEdit extends Component {
...
+
   componentDidUpdate(previousProps, previousState) {
     const {
       collection,
-      dispatch,
       documentId,
       group,
-      state
+      state,
+      actions
```
Comment (mingard) [inline frontend/containers/DocumentEdit/DocumentEdit.jsx]: Not sure why this wasn't caught by linter

File: frontend/containers/DocumentEdit/DocumentEdit.jsx (hunk 1 above)
Comment (eduardoboucas) [inline frontend/containers/DocumentEdit/DocumentEdit.jsx]: This one is resolved!

File: frontend/containers/App/App.jsx [hunk 4]
```diff
@@ -23,11 +24,29 @@ import SignOutView from 'views/SignOutView/SignOutView'
...
+  constructor (props) {
+    super(props)
+    const {actions, state} = this.props
+
+    this.socket = new Socket()
+      .on('userListChange', this.handleUserListChange.bind(this))
+  }
+
+  handleUserListChange (data) {
+    const {state, actions} = this.props
```
Comment (eduardoboucas) [inline frontend/containers/App/App.jsx]: This is resolved.

**Pull request merged**